main_logger.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"github.com/rond-authz/rond/internal/config"
+
+	"github.com/sirupsen/logrus"
+)
+
+type failLogger struct {
+	log *logrus.Logger
+	env config.EnvironmentVariables
+}
+
+func (failLogger failLogger) opaModulesLoad(err error) {
```

Comment:
Why do you think this is a good way of incapsulating logs? There is no actual layer of abstraction, only more code just to produce a log 🤔

Reply:
Because the main file was too verbose, I think that `main.go` should not care how messages are logged, but just to log them.
This new file is made with this purpose: define HOW to log them.

Reply:
well, we abstract the logger and the source of environment variables.
Instead of repeating them in all the `main` source file, we use them once just to create the `failLogger` struct.

After that, they are "hidden" and not repeated anymore.

Reply:
there is no repetition for accessign env, each log simply uses the logger instance created at start 🤔 

env is only accessed in the first two of five logs

Reply:
Code roll-backed